src/core/opentelemetry.ml

```diff
@@ -855,6 +880,14 @@ module Scope : sig
     unit ->
     t
 
+  val to_span_link :
+    ?trace_state:string ->
+    ?attrs:key_value list ->
+    ?dropped_attributes_count:int ->
+    t ->
+    Span_link.t
+  (** Turn the scope into a span link *)
```

Comment:
I'm currently doing the following in a codebase to get the `span_link`

`scope |> Scope.to_span_ctx |> Span_link.of_span_ctx`

Is the `Span_link.of_span_ctx` still required now that we can go directly from `scope` to `span_link` ?

Reply:
And if yes, should we add `?dropped_attributes_count:int` to the `Span_link.of_span_ctx` to be consistent ?

Reply:
I think we can keep both, but yes, adding the optional arguments is good for consistency.